databricks/koalas/accessors.py

```diff
@@ -557,21 +557,27 @@ def pandas_concat(series):
             # The input can only be a DataFrame for struct from Spark 3.0.
             # This works around to make the input as a frame. See SPARK-27240
             pdf = pd.concat(series, axis=1)
-            pdf = pdf.rename(columns=dict(zip(pdf.columns, names)))
+            pdf.columns = names
             return pdf
 
+        def apply_func(pdf):
+            return func(pdf).to_frame()
+
         def pandas_extract(pdf, name):
             # This is for output to work around a DataFrame for struct
             # from Spark 3.0.  See SPARK-23836
             return pdf[name]
 
-        def pandas_series_func(f):
+        def pandas_series_func(f, by_pass):
```

Comment:
Why is it called `by_pass`? Would you please help me understand?

Reply:
It uses some new Spark APIs to "by pass" a workaround:

You can see:
- https://github.com/databricks/koalas/blob/80a5893116401b3059c896e96c59d509c82ca025/databricks/koalas/accessors.py#L646-L666
- https://github.com/databricks/koalas/blob/80a5893116401b3059c896e96c59d509c82ca025/databricks/koalas/accessors.py#L715-L735